pkgs/os-specific/linux/purism/librem-ec-acpi/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  kernel,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "librem-ec-acpi";
+  version = "0.9.2-${kernel.version}";
+
+  src = fetchFromGitLab {
+    domain = "source.puri.sm";
+    owner = "nicole.faerber";
+    repo = "librem-ec-acpi-dkms";
+    rev = "cec8c0e8bf1532c9c605f60bb02a2ef0f98e5d77";
+    sha256 = "gPb1/4UPbwjJiqtP27KKjiIKxIYEl2nlQACUAlxVEFU=";
```

Comment:
Nit: per https://nixos.org/manual/nixpkgs/stable/#sec-pkgs-fetchers-fetchurl-inputs it's generally preferred to provide this as a `hash` attribute.